package.json

```diff
@@ -16,7 +16,8 @@
     "pretest": "npm run lint",
     "test": "lerna run test",
     "posttest": "lerna run --scope @turf/turf last-checks",
-    "docs": "node ./scripts/generate-readmes"
+    "docs": "node ./scripts/generate-readmes",
+    "coverage": "node ./scripts/coverage"
```

Comment:
It looks like you added a lot of the package.json changes manually. You could've used `.monorepolint.config.js` to both apply these and enforce it going forward.

Reply:
Ah! I didn't realise this was how this works - still getting my head around all the monorepo scaffolding...

Reply:
Yeah monorepolint is something one of my coworkers wrote. Its great but also nobody really knows about it.